### ISSUE TITLE
[Feature] Disable Merging Window Modality

### ISSUE THREAD

**Issue by kpTed** [ForkIssues/TrackerWin#1888]
Actually, the Merging Window is a modal window (block focus on the merging windows), it would be very useful to check all commits history in the main window to help resolve conflicts.

**Comment by Srekel** [comment]
Yes, being able to inspect history while resolving a merge conflict is super important :)
